frame/evm/src/lib.rs

```diff
@@ -137,12 +140,43 @@ pub trait Trait: frame_system::Trait + pallet_timestamp::Trait {
 	}
 }
 
+#[derive(Clone, Eq, PartialEq, Encode, Decode)]
+#[cfg_attr(feature = "std", derive(Debug, Serialize, Deserialize))]
+/// Account definition used for genesis block construction.
```

Comment:
```suggestion
/// Account definition used for genesis block construction.
#[derive(Clone, Eq, PartialEq, codec::Encode, codec::Decode, Debug, serde::Serialize, serde::Deserialize)]
#[cfg(feature = "std")]
```

Genesis types are only required in native.